clients/runtime/src/types.rs

```diff
@@ -114,6 +115,24 @@ mod metadata_aliases {
 	}
 }
 
+pub mod currency_id {
+	use super::*;
+	use crate::Error;
+
+	pub trait CurrencyIdExt {
+		fn inner(&self) -> Result<u8, Error>;
+	}
+
+	impl CurrencyIdExt for CurrencyId {
+		fn inner(&self) -> Result<u8, Error> {
+			match self {
+				CurrencyId::XCM(x) => Ok(*x),
+				_ => Err(Error::CurrencyNotFound),
```

Comment:
This needs to implement something for our `CurrencyId::Stellar` types as well.

Reply:
We should consider not returning an `u8` as a result here but a string or a Vec<> maybe. u8 is not enough to provide a meaningful description.

Reply:
Note that this is used to describe the currency in our metric reporting so it should be unambiguous

Reply:
@ebma is it ok to use debug formatter for CurrencyId?